lattices/README.md

```diff
@@ -0,0 +1,66 @@
+# `lattices`
+
+<!-- ANCHOR: book_include -->
+
+The `lattices` crate provides traits and implementations for semilattice data types. Semilattices
+are the mathematical concept underyling CRDTs;
+their useful properties (associativity, commutativity, and idempotence) allow data updates to run
+in any order while still preserving the same end result.
```

Comment:
I think 'allow data updates to run in any order while still preserving the same end result' is really underselling lattices. Maybe we should talk about monotonicity here, and also about what the profound implications of all of these properties are, like how it provides a principled way to build distributed systems or something to that effect.

This is the first thing in readme, so I think this should have some good marketing hook text. Some kind of why does this crate exist thing, why it's so amazing, etc.